### DISCUSSION TITLE
Fixed bug with prepare_C2M2_submission.py script

### DISCUSSION THREAD

**Discussion by jessicalumian** [nih-cfde/published-documentation#348] [Known Issues] [answered]
There was an issue with the prepare_C2M2_submission.py script that was released with the update from this past Monday. A new field in the file dbgap_study_id file was not present as expected and would result in an error.

**This issue has since been fixed! If you have this problem, please download the new script here: https://osf.io/bq6k9/**

**Comment by ACharbonneau** [comment, marked as answer]
This issue has since been fixed! If you have this problem, please download the new script here: https://osf.io/bq6k9/
